src/cascadia/TerminalApp/CascadiaSettings.cpp

```diff
@@ -63,6 +63,29 @@ CascadiaSettings::CascadiaSettings(const bool addDynamicProfiles)
     }
 }
 
+// Method Description:
+// - Finds a GUID associated with the given profile name. If no profile matches
+//      the profile name, returns a null GUID.
+// Arguments:
+// - profileName: the name of the profile's GUID to return.
+// Return Value:
+// - the GUID associated with the profile name.
+GUID CascadiaSettings::FindGuid(const std::wstring& profileName) const noexcept
+{
+    GUID profileGuid{};
+
+    for (const auto& profile : _profiles)
+    {
+        if (profileName == profile.GetName())
+        {
+            profileGuid = profile.GetGuid();
+            break;
+        }
+    }
+
+    return profileGuid;

```

Comment:
I think my only real comment is that I think I'd prefer that we make this function return a `std::optional<GUID>`, and return `std::nullopt` when we didn't find a match instead of returning the null GUID, since the null GUID _is technically a valid GUID_.

Reply:
Sure thing